portality/tasks/reporting.py

```diff
@@ -120,7 +121,7 @@ def _tabulate_time_entity_group(group, entityKey):
 
 def _fft(timestamp):
     """File Friendly Timestamp - Windows doesn't appreciate : / etc in filenames; strip these out"""
-    return dates.reformat(timestamp, app.config.get("DEFAULT_DATE_FORMAT"), "%Y-%m-%d")
+    return dates.reformat(timestamp, dates.FMT_DATETIME_STD, FMT_DATE_STD)
```

Comment:
so minor barely worth mentioning, but why `dates.FMT_DATETIME_STD` but actually importing `FMT_DATE_STD`?  Mixed approach seems odd, go for one or the other

Reply:
because the replace string of `app.config.get("DEFAULT_DATE_FORMAT")` is `dates.FMT_DATETIME_STD`
and the replace string of `"%Y-%m-%d"` is `FMT_DATE_STD`

when replacing it replace all file, so I have not customise for any specify file.